rotkehlchen/tests/unit/decoders/test_aerodrome.py

```diff
@@ -530,3 +531,81 @@ def test_vote(base_accounts, base_transaction_decoder):
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize('load_global_caches', [[CPT_AERODROME]])
+@pytest.mark.parametrize('base_accounts', [['0x7264A62ae2f2BbE5Fe003F29108afB3C3dA0Bc16']])
+def test_swap(base_transaction_decoder, base_accounts, load_global_caches):
+    evmhash = deserialize_evm_tx_hash('0x260538961f3f2e17a43d26732f1105f739f9cf79622a3df8986c279c6d69a450')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=base_transaction_decoder.evm_inquirer,
+        tx_hash=evmhash,
+        load_global_caches=load_global_caches,
+    )
+    expected_events = [
+        EvmEvent(
```

Comment:
```suggestion
    assert events == [EvmEvent(
```
can make this more compact